tests/compile-fail/attr_on_enum.stderr

```diff
@@ -1,9 +1,9 @@
 error: couldn't parse trait item
 
-note: expected `trait`
+  = note: expected `trait`
+  = note: the #[auto_impl] attribute can only be applied to traits!
 
-note: the #[auto_impl] attribute can only be applied to traits!
- --> $DIR/attr_on_enum.rs:5:1
+ --> $DIR/attr_on_enum.rs:4:1
   |
-5 | enum Foo {
-  | ^^^^
+4 | #[auto_impl(&, &mut)]
+  | ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Mhhh, I don't particularly like this span, but I agree that it's difficult to say what span would be the best for this error. Well, I guess it's fine like this.